pom.xml

```diff
@@ -29,7 +29,7 @@
         <springLdapVersion>2.3.2.RELEASE</springLdapVersion>
         <springSecurityVersion>5.0.6.RELEASE</springSecurityVersion>
         <springSessionVersion>2.0.2.RELEASE</springSessionVersion>
-        <forkCountTests>2C</forkCountTests>
+        <forkCountTests>1</forkCountTests>
```

Comment:
Is the fork count supposed to be changed?

Reply:
yes